setup.cfg

```diff
@@ -34,8 +34,8 @@ python_requires = >=3.7
 install_requires =
     scikit-learn>=1.0
     nflows>=0.14
-    pandas>=1.3,<2.0
```

Comment:
this will cause some crashes atm.

Reply:
Do you mean I should add `<2.0` at the end?

Reply:
Yes, the tests will crash otherwise. `lifelines` doesn't support pandas2, and asdgan/ctgan crash for some reason from torch2

Reply:
I have updated